katello.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "gettext_i18n_rails"
 
   # Pulp
-  gem.add_dependency "runcible", ">= 1.3.0"
+  gem.add_dependency "runcible", ">= 1.6.0"
```

Comment:
Would you mind adding a major version limit while you are at it? Similar to https://github.com/Katello/katello/pull/5772/files#diff-1cd74c6b4bb8c448bf94e0aa73b64f1bR40